apps/base-docs/docs/connecting-to-the-blockchain/configuring-providers-sbs.md

```diff
@@ -0,0 +1,235 @@
+
+Before you can do anything else, you need to obtain a _WalletConnect_ `projectId`.
+
+Open up the [WalletConnect] homepage, and create an account and/or sign in using the method of your choice.
+
+Click the `Create` button in the upper right of the `Projects` tab.
+
+![Create Button](../../assets/images/connecting-to-the-blockchain/wallet-connect-create-button.png)
+
+Enter a name for your project, select the `App` option, and click `Create`.
+
+![Project Information](../../assets/images/connecting-to-the-blockchain/add-project-information.png)
+
+Copy the _Project ID_ from the project information page, and paste it in as the `projectId` in `getDefaultWallets`.
+
+```typescript
+const { connectors } = getDefaultWallets({
+  appName: 'RainbowKit App',
+  projectId: 'YOUR_PROJECT_ID',
+  chains,
+});
+```
+
+:::caution
+
+Remember, anything you put on the frontend is public! That includes this id, even if you use environment variables to better manage this type of data. Next.js reminds you of the risk, by requiring you to prepend `NEXT_PUBLIC_` to any environment variables that can be read by the browser.
+
+Before you deploy, make sure you configure the rest of the items in the control panel to ensure only your site can use this id.
+
+:::
+
+### Public Provider
+
+By default, the setup script will configure your app to use the `publicProvider()`, and connect to a number of popular chains. To simply matters, remove all but `mainnnet`, `optimism`, and `base`.
+
+```typescript
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet, optimism, base],
+  [publicProvider()],
+);
+```
+
+Open the terminal and start the app with:
+
+```bash
+yarn run dev
+```
+
+Click the `Connect Wallet` button, select your wallet from the modal, approve the connection, and you should see your network, token balance, and address or ENS name at the top of the screen. Select your wallet from the modal.
+
+![Rainbowkit modal](../../assets/images/connecting-to-the-blockchain/rainbowkit-modal.png)
+
+You've connected with the Public Provider!
+
+![Connected](../../assets/images/connecting-to-the-blockchain/connected.png)
+
+### QuickNode
+
+Notice that the [`configureChains`] function takes in an array of providers. It will attempt to use each one in the order provided, falling back to the next in the list in order if something goes wrong. You can use this for redundancy, or to smoothly handle scenarios where your app needs to connect with a chain that isn't supported by your preferred provider.
+
+Let's add [QuickNode] to the list. It isn't [included in the library] by default, so you'll need to add it using the generic [JSON RPC provider] import.
+
+```typescript
+import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
+```
+
+Unlike the linked example indicates, you do **not** need to add an import `chain` from `'wagmi'` (Their example appears out of date at the time of writing).
```

Comment:
HA! Of course...

Reply:
But the language is a little harsh.  Updated!